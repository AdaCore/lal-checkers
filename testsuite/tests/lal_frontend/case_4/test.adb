procedure Test is
   x : Integer;
   y : Integer;
begin
   case x is
      when Natural =>
         y := 43;
      when others =>
         y := x;
   end case;
end Ex1;
