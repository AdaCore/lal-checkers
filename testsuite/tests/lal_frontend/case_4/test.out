Program:
  x#1 = Input()
  y#2 = Input()
  split:
    assume(&&(>=(x#1, 0), <=(x#1, 2147483647)))
    y#2 = 43
  |:
    assume(!(&&(>=(x#1, 0), <=(x#1, 2147483647))))
    y#2 = x#1
  end:
